stylesheets/_layout.settings.scss

```diff
@@ -467,3 +467,101 @@
 .content-actions__buttons .btn {
     vertical-align: top;
 }
+
+/**
+ * Stacked inputs
+ */
+ .tab__settings .settings-section--stacked {
+
+    .form__group {
+        clear: both;
+    }
+
+    // Force labels to be stacked with the inputs below, except for checkbox/radio
+    .form__group:not(.form__group--flush):not(.form__group--toggle):not(.form-checkbox):not(.form-radio) {
+        margin-bottom: 12px;
+
+        > .control__label {
+            padding-top: 0;
+        }
+
+        > .control__label,
+        .controls,
+        .form__control:not(.checkbox):not(.radio),
+        .form__control + .select2,
+        .input-group {
+            width: 100%;
+        }
+
+        .ie9 &.colorpicker .form__control {
+            width: calc(100% - 48px);
+        }
+
+        .help-block {
+            margin: 0;
+        }
+    }
+
+    // Because the settings sidebar and the toggle switch controls are a fixed width, we're safe to use a fixed width here
+    .form__group--toggle {
+        .controls {
+            float: left;
+
+            @include respond-min($screen-medium) {
+                float: right;
+            }
+        }
+
+        .control__label {
+            margin-left: 10px;
+            width: 80%;
+
+            @include respond-min($screen-smaller) {
+                width: 125px;
+            }
+
+            @include respond-min($screen-medium) {
+                margin-left: 0;
+                width: 250px;
+            }
+        }
+    }
+
+    // Instead of stacking, keep checkbox on left edge
+    .form__group.form-checkbox,
+    .form__group.form-radio {
+        position: relative;
+
+        > .control__label {
+            padding-left: 30px;
+            min-width: 300px;
```

Comment:
There's prbably a reason for this but do we have to use px based widths here rather than %? same for the line below.

Reply:
The settings sidebar a fixed `px` width